projects/sandbox/src/app/dataset-info/dataset-info.component.ts

```diff
@@ -37,6 +37,12 @@ export class DatasetInfoComponent extends SubscriptionManager {
       this.sandbox
         .getDatasetInfo(datasetId, this.status !== DatasetStatus.COMPLETED)
         .subscribe((info: DatasetInfo) => {
+          let creationDate = info['creation-date'];
+          info['creation-date'] = formatDate(
+            this.convertUTCDateToLocalDate(creationDate),
+            'dd/MM/yyyy, HH:mm:ss',
+            'en-GB'
```

Comment:
Is it correct to simply assume that it is always this locale?

Reply:
The convert function assumes a non-localised / universal date (i.e. suffixed with something like the `+01:00` instead of the `Z`) as a parameter.  The 3rd parameter (`en-GB`) specifies the UI locale.